terraform/modules/ci/main.tf

```diff
@@ -91,6 +91,11 @@ resource "google_cloud_run_service" "cloudlogging_backend_client_services" {
   location = var.region
 
   template {
+    metadata {
+      annotations = {
+        "run.googleapis.com/cpu-throttling" = "false"
```

Comment:
Another option we can look into is using something like micronaut for faster JVM startups. Really depends on what deployment patterns we expect. Spring is more popular but has a much more expensive cold start for serverless workflows.

Reply:
Good to know, could be a topic in the future if we want to build java apps! Lumberjack creates a java client using Guice, it does not run a java app, the java app running here is a test app just for CI, should not be a concern.